frontends/election-manager/src/screens/logic_and_accuracy_screen.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { LinkButton } from '@votingworks/ui';
+
+import { NavigationScreen } from '../components/navigation_screen';
+import { Prose } from '../components/prose';
+import { routerPaths } from '../router_paths';
+import { ZeroReportPrintButton } from '../components/zero_report_print_button';
+
+export function LogicAndAccuracyScreen(): JSX.Element {
+  return (
+    <NavigationScreen>
+      <Prose maxWidth={false}>
+        <h1>L&amp;A Materials</h1>
+        <p>
+          <ZeroReportPrintButton />
+        </p>
+        <p>
+          <LinkButton to={routerPaths.printTestDecks}>
+            Print Test Decks
+          </LinkButton>{' '}
+          <LinkButton to={routerPaths.testDecksTally}>
+            View Test Ballot Deck Tally
```

Comment:
Good observation. I'd just copied the text from the Tally tab but agree that we should standardize. Will pass through and clean things up 🧹

Reply:
Yeah, let's update "View Test Ballot Deck Tally" to "Print Test Deck Tally".

Reply:
Also, can we put the buttons each on their own line now that there is more room on the page?

Reply:
https://github.com/votingworks/vxsuite/pull/1708/commits/df5146183642b4f921841739c35b80719bdcbe5c + https://github.com/votingworks/vxsuite/pull/1708/commits/82f931d91024ffc67f8bea11fa9aa8f2837a00f7

Went with "Print Test Deck Tally Reports" instead of "Print Test Deck Tally" for further consistency with other pages